app/models/product.rb

```diff
@@ -24,6 +24,14 @@ class Product < ApplicationRecord
       all
     end
   }
+  scope :by_status, lambda {|status|
+    if status ==
+       Settings.product.status.all
+      all
+    else
+      where(status: status)
```

Comment:
```suggestion
      where status: status if status.present? 
```
ri là dc mà em cần j if else ở đây